client/src/components/Instructions/index.js

```diff
@@ -92,6 +92,100 @@ export default class Instructions extends Component {
     );
   }
 
+  renderGaslessCounterSetup() {
+    const addressDefault = this.props.ganacheAccounts.length > 2 ? this.props.ganacheAccounts[2] : '<ADDRESS>';
+    return (
+      <div className={styles.instructions}>
+        <h2> Build your first app with ZepKit </h2>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            1. Ensure Docker is installed on your computer.
+          </div>
+          <div className={styles.code}>
+            <code>
+              <a href="https://docs.docker.com/install/">https://docs.docker.com/install/</a>
+            </code>
+          </div>
+        </div>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            2. In another terminal, run the gas relay docker container with ganache, wait a few seconds for it to sping up (run inside your zepkit folder).
+          </div>
+          <div className={styles.code}>
+            <code>
+              npm explore tabookey-gasless npm run gsn-dock-relay-ganache
+            </code>
+          </div>
+        </div>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            3. In a different terminal, init your project with ZeppelinOS (run inside your zepkit folder).
+          </div>
+          <div className={styles.code}>
+            <code>
+              zos init zepkit
+            </code>
+          </div>
+        </div>
+        <div className={styles.step}>
+          <div className={styles.instruction}>
+            4. Add the GaslessCounter contract to your project (Note that a compilation warning will be thrown, you can ignore this).
```

Comment:
I didn't get any compiler warnings here (I'm using truffle 5.0.4). What were you getting? Maybe we should fix the solc version in truffle-config to avoid these?

Reply:
This was left over from an older commit, fixed this warning, thanks for pointing it out, removed in next commit :)